packages/commonwealth/client/scripts/views/pages/ComponentsShowcase/components/Cards.showcase.tsx

```diff
@@ -32,6 +34,16 @@ const CardsShowcase = () => {
         header="Information"
         content={<CWText className="content">Content page card content</CWText>}
       />
+
+      <CWText type="h5">Related Community Card</CWText>
+      <CWRelatedCommunityCard
+        community={app.config.chains.getAll()[30]}
+        connected={false}
```

Comment:
Since the `connected` prop is a optional boolean and we're hard setting it here. The absence of the prop would be falsey not need to include this